### PR TITLE
Fix types in message and misk

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -100,6 +103,7 @@
     "moment": "2.22.2",
     "node-sass": "4.11.0",
     "node-sass-importer": "1.0.0",
+    "npm": "^6.7.0",
```
Comment (fredj) [inline package.json]: is it correct to have npm here ?

File: package.json [hunk 2]
```diff
@@ -79,6 +81,7 @@
     "fs-extra": "7.0.1",
     "html-webpack-include-assets-plugin": "1.0.6",
     "html-webpack-plugin": "3.2.0",
+    "install": "^0.12.2",
```
Comment (fredj) [inline package.json]: ?

File: package.json (hunk 1 above)
Comment (sbrunner) [reply package.json]: ?

File: package.json (hunk 2 above)
Comment (sbrunner) [reply package.json]: ???

**Pull request merged**